helm/athena/values.yaml

```diff
@@ -7,6 +7,9 @@ image:
   repository: giantswarm/athena
   tag: "[[.Version]]"
 
+clusterID: ""
```

Comment:
What is this? And should it maybe be `clusterName`?

Reply:
We need a name for the cluster to be returned by athena. Currently `managementCluster.name` is used but I wanted to add an option that is not confusing when we install on a workload cluster.
`clusterID` and `baseDomain` are standard values for our WC apps like here
https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/values.yaml#L430-L438
AFAIK they are set automatically so it makes sense to use them.

Reply:
ACK, thanks